.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: Feature Request
+description: Suggest an idea for this project
+labels: [feature-request]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a feature request against this repo, consider whether the feature should/could be implemented in the [other OpenTelemetry client libraries](https://github.com/open-telemetry/). If so, please [open an issue on opentelemetry-specification](https://github.com/open-telemetry/opentelemetry-specification/issues/new) first.
+  - type: textarea
+    id: related-problem
+    attributes:
+      label: Is your feature request related to a problem?
```

Comment:
I would rephrase it to: What problem do you want to solve?

Reply:
Excellent. I liked it. When we have the final review here, I'll fix this in the main repo, too to keep forms similar.